server/utils/auth/google_client.py

```diff
@@ -120,4 +120,14 @@ def create_event_type(self, duration, title, start_time, timezone, host_email, b
 
         event_service = build('calendar', 'v3', credentials=self.credentials)
         event = event_service.events().insert(calendarId='primary', body=event).execute()
-        # print('Event created: %s' % (event.get('htmlLink')))
\ No newline at end of file
+        return event.get("id")
+        # print('Event created: %s' % (event.get('htmlLink')))
+    
+    def delete_calendar_event(self,google_event_id):
+        try:
+            service = build('calendar', 'v3', credentials=self.credentials)
+            service.events().delete(calendarId='primary', eventId='eventId').execute()
+            return True
```

Comment:
Can probably return something more meaningful if we have time to refactor

Reply:
Just come out with this:
![image](https://user-images.githubusercontent.com/70559690/113094064-e60a5400-91a5-11eb-8b51-f9b57fb45a95.png)

and ready to do this:
![image](https://user-images.githubusercontent.com/70559690/113094104-f7ebf700-91a5-11eb-988c-50ceccc438e6.png)

gonna push soon